### PR TITLE
Makefile target for versioning

### PR DESCRIPTION
**Pull request by remenska**

### CODE REVIEW

File: src/Makefile [hunk 1]
```diff
@@ -19,26 +19,45 @@ I18NSPHINXOPTS  = $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) source
...
+	mkdir -p ../doc/$(v)
+	cp -r $(WEBAPPDIR)/ ../doc/$(v)/
+	rm ../doc/prod
+	ln -s $(v) ../doc/prod
+	rm ../doc/sandbox
+	ln -s $(v) ../doc/sandbox
+	@echo
+	@echo "Versioning finished. New version is $(v)"
+
+check-version:
```
Comment (twessling-icas) [inline src/Makefile:61]: is there an option to also let the check look at the version in conf.py ?

**Review by twessling-icas — DISMISSED** [review]

File: src/Makefile (hunk 1 above)
Comment (remenska) [reply src/Makefile:61]: good idea, bit far fetched but let me see what I can do :P

Comment (remenska) [reply src/Makefile:61]: done!

**Review by dkocic — APPROVED** [review]
Yes

**Pull request merged**